.gitignore

```diff
@@ -226,3 +226,6 @@ CVS
 .ssh/*
 id_rsa*
 id_dsa*
+
+# Carthage
+Carthage/Build
```

Comment:
The issue you linked to at the start of this PR: Carthage/Carthage#1306 specified to add `Carthage/Build/` rather than `Carthage/Build`. The .gitignore already had `build/` with the comment the slash was there to avoid collisions with other files that begin with `build`. Including the slash seems like the safer bet.